packages/web/src/components/layouts/PackagePage/PackagePage.tsx

```diff
@@ -0,0 +1,596 @@
+                        >
+                          <Icon kind='link' color='#8E8AA0' className={styles.externalLinkIcon} />
+                          Homepage
+                        </a>
+                      </>
+                    )}
+                  </div>
+                </div>
+
+                {/* TODO: probably hide about section on mobile with non-css solution */}
+                <div className={clsx(styles.sidebarItem, styles.sidebarItemDesktop)}>
+                  {loading ? (
+                    <>
+                      <div className={styles.sidebarItemTitle}>
+                        <Skeleton width={49} />
+                      </div>
+                      <Skeleton width='90%' />
+                      <Skeleton width='50%' />
+                    </>
+                  ) : (
+                    <>
+                      <div className={styles.sidebarItemTitle}>About</div>
+                      <p className={styles.sidebarItemText}>
+                        A JavaScript date library for parsing, validating, manipulating, and
+                        formatting dates
+                      </p>
+                      <button className={styles.link}>Go down to full description</button>
+                    </>
+                  )}
+                </div>
+
+                <div className={styles.sidebarItem}>
+                  <div className={styles.stats}>
+                    {loading ? (
+                      repeat(
+                        3,
+                        <div className={styles.stat}>
+                          <Skeleton
+                            width={52}
+                            height={52}
+                            variant='circular'
+                            className={styles.statImagePlaceholder}
+                          />
+                          <div className={styles.statContent}>
+                            <span className={styles.statTitle}>
+                              <Skeleton width={49} />
+                            </span>
+                            <span className={styles.statValue}>
+                              <Skeleton width={108} />
+                            </span>
+                          </div>
+                        </div>
+                      )
+                    ) : (
+                      <>
+                        <div className={styles.stat}>
+                          <div className={styles.statImagePlaceholder}>
+                            <img
+                              className={styles.statImage}
+                              src='https://via.placeholder.com/52'
+                              alt=''
+                            />
+                          </div>
+
+                          <div className={styles.statContent}>
+                            <span className={styles.statTitle}>Author</span>
+                            <span className={styles.statValue}>jdalton</span>
```

Comment:
And here.
Also, what about more than one author? I think what you did in `PackagePreview` with multiple avatars looks great, maybe we can use it here too.